### PR TITLE
J0hnby dhcp server ntp

### PR DESCRIPTION
**Pull request by j0hnby**
Modified config to bump to ver 1.3 and have ntp options and schema
Modified run.sh to include ntp servers from config if they exist
Modified docs and changelog to include additional detail.

Testing performed with local repo, started with current version, made changes, incremented version made changes as here, tested to ensure defaults arrive with upgrade, then tested with ntp servers specified and not-specified.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @j0hnby, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#2376).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: dhcp_server/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3
```
Comment (pvizeli) [inline dhcp_server/CHANGELOG.md:3]:
```suggestion
## 1.3.0
```

File: dhcp_server/config.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "1.2"
+version: "1.3"
```
Comment (pvizeli) [inline dhcp_server/config.yaml:1]:
```suggestion
version: 1.3.0
```

**Review by pvizeli — CHANGES_REQUESTED** [review]

**Review by mdegat01 — APPROVED** [review]
@j0hnby Please rebase your branch and resolve the conflict with `config.yaml` (looks like you just need to add `---` to the top as a formatter added that). Then I will merge this.

**Comment by pvizeli** [comment]
@j0hnby please address the lint issue

**Pull request merged**